### PR TITLE
BaSP-MR-209: Created shared side bar and logic for display different views

### PR DESCRIPTION
**Pull request by fernandofernandezuy**
In collaboration with Lorenzo Mauro, Alejandro Macia, Gonzalo Reybet and Daniel Ninfi

### CODE REVIEW

**Comment by AleMac17** [comment]
[[Sidebar] Functionality](https://trello.com/c/HrhlDtsg/209-sidebar-functionality)

File: src/Components/Header/header.module.css [hunk 1]
```diff
@@ -24,6 +24,14 @@
   margin-right: 1rem;
 }
 
+.logOut {
+  width: 24px;
+  height: 24px;
+  margin-left: 5px;
+  margin-right: 1rem;
+background-color: red;
```
Comment (AleMac17) [inline src/Components/Header/header.module.css]: .

**Review by EmaBaldor — APPROVED** [review]
Excellent PR! Very complete.

**Comment by nazarenapantanetti** [comment]
Tested in master, great job!!

**Pull request merged**